site-new/gen-src-temp/versions.json

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
The files under `/gen-src-temp` are to be removed after migrating `build.gradle`.